.github/workflows/unit_tests.yml

```diff
@@ -6,5 +6,26 @@ on:
 jobs:
   py_build_tests:
     uses: neongeckocom/.github/.github/workflows/python_build_tests.yml@master
-    with:
-      python_version: "3.8"
+  unit_tests:
+    strategy:
+      matrix:
+        python-version: [ 3.9, "3.10", "3.11", "3.12" ]
```

Comment:
we do not intend to support Python 3.8 in our system anymore?

Reply:
I've been dropping it from tests as it [recently reached EoL](https://devguide.python.org/versions/)